### PR TITLE
Set the Source.uid in API spec tests

### PR DESCRIPTION
**Pull request by agrare**
https://github.com/ManageIQ/topological_inventory-core/pull/74 made the source uid mandatory so update the API specs accordingly.

### CODE REVIEW

File: app/controllers/api/v0/sources_controller.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def create
       private
 
       def create_params
-        body_params.permit(:name, :source_type_id, :tenant_id)
+        body_params.permit(:name, :source_type_id, :tenant_id, :uid)
```
Comment (carbonin) [inline app/controllers/api/v0/sources_controller.rb]: Should we really be having the client give us a uid? Would it be better to generate one on the model?

Comment (agrare) [reply app/controllers/api/v0/sources_controller.rb]: I was thinking there might be cases where we're using the native provider uuid but that doesn't apply to any of these providers yet so yeah i'll just generate it

Comment (carbonin) [reply app/controllers/api/v0/sources_controller.rb]: Does this line still need to be here? Or can this be removed? It looks like we'll overwrite anything that is passed in the params based on the `.merge!` above, right?

Comment (agrare) [reply app/controllers/api/v0/sources_controller.rb]: Yup just forgot to delete it

**Comment by bdunne** [comment]
@agrare I don't think this should be here, is there a follow-up to the core repo to set this with `default_value_for` or something else on the model?

**Pull request merged**